src/components/ui/ColorButtons.scss

```diff
@@ -0,0 +1,33 @@
+.colorbuttons{
```

Comment:
If we follow the global css pattern (which i don't like), this file should be in src/styles/ui.
I would prefer the component based approach with a component ColorButton which has a parameter vairant (which controls the color), and RulesButton uses the ColorButton and chooses one of the variants.

Reply:
We can talk about this during our next meeting